configure.ac

```diff
@@ -61,6 +61,21 @@ PKG_CHECK_MODULES(GST, [
   ])
 ])
 
+dnl check for host OS
+AC_CANONICAL_HOST
+
+case "${host_os}" in
```

Comment:
if `host_os` is not one of these three options? Totally fail or continue as unknown host_os?

Reply:
Continue with unknown OS. The CPU is printed with -1.